sanitize/sanitize_change.go

```diff
@@ -10,18 +10,12 @@ import (
 // SanitizeChange traverses a Change and replaces all values at
 // the particular locations marked by BeforeSensitive AfterSensitive
 // with the value supplied as replaceWith.
-//
-// A new change is issued.
-func SanitizeChange(old *tfjson.Change, replaceWith interface{}) (*tfjson.Change, error) {
-	result, err := copyChange(old)
-	if err != nil {
-		return nil, err
+func SanitizeChange(result *tfjson.Change, replaceWith interface{}) {
+	if result == nil {
```

Comment:
this should be handled in the caller:
```
if res == nil {
  return
}
SanitizeChange(res, replace)
```
or
```go
if res != nil {
    SanitizeChange(res, replace)
}
```

otherwise it's too easy to forget those early checks in some functions leading to panics.